portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +9,82 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.active {
+    background-color: #008080;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: black;
+    opacity: 30;
```

Comment:
Optional: At Google, we organize style attributes alphabetically, so you would have background-color, list-style-type, margin, opacity, overflow, padding

Up to you if you want to change this, here and below. Alphabetically means its easier to find what you're looking for, but also sometimes some attributes work well together/only make sense together, so you can also group them like that